diff -ur phpldapadmin-0.9.8.2-orig/htdocs/index.php phpldapadmin-0.9.8.2/htdocs/index.php
--- phpldapadmin-0.9.8.2-orig/htdocs/index.php	2006-03-08 22:49:27.000000000 +0000
+++ phpldapadmin-0.9.8.2/htdocs/index.php	2006-04-05 20:24:11.795075500 +0000
@@ -18,7 +18,7 @@
  * We will perform some sanity checking here, since this file is normally loaded first when users
  * first setup PLA.
  */
-define('LIBDIR','../lib/');
+define('LIBDIR','lib/');
 ini_set('display_errors',1);
 error_reporting(E_ALL);
 
Only in phpldapadmin-0.9.8.2/htdocs: index.php~
